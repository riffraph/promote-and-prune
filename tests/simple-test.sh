#!/bin/sh

TESTDATAFOLDER=simpleTestDataset

# simple test

mkdir -p $TESTDATAFOLDER/folder1

head -c 1024 /dev/urandom > $TESTDATAFOLDER/folder1/file1

identify.sh $TESTDATAFOLDER 1k
identify.sh $TESTDATAFOLDER 1k | promote.sh


#rm -rf $TESTDATAFOLDER