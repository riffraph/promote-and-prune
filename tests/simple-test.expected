8 simpleTestDataset/folder1/file1
simpleTestDataset/file1
simpleTestDataset/folder1
simpleTestDataset
